### PR TITLE
Create integration.md

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

This PR includes the following proposed change(s):

None yet

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Via PR

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/integration.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+## Git Action Workflow
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://app.lucidchart.com/documents/embeddedchart/cd317d74-4127-4518-8ad9-b99e7a66e005" id="M9w_SX3Zmq.f"></iframe></div>
+
+## Code Climate
+
+The JAG File Submission project uses (Code Climate)[codeclimate.com] as its code quality tool. Much like SonarQube, Code Climate reads and interprets the code coverage reports generated by both the frontend and backend components of this project, and then returns a Pass/Fail status back to Github. This is an attempt to move the code quality gate in this projects development lifecycle to a point where the Maintainability and Test Coverage scores are as high as possible before this application reaches a production ready state.
```
Comment (sdevalapurkar-bcgov) [inline docs/integration.md:8]:
```suggestion
The JAG File Submission project uses (Code Climate)[codeclimate.com] as its code quality tool. Much like SonarQube, Code Climate reads and interprets the code coverage reports generated by both the frontend and backend components of this project, and then returns a Pass/Fail status back to Github. This is an attempt to move the code quality gate in this project's development lifecycle to a point where the Maintainability and Test Coverage scores are as high as possible before this application reaches a production ready state. This is enabled by having a strict quality gate enforced on the pull request level prior to code being merged into the master branch.
```

File: docs/integration.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+View the JAG File Submission code quality report (here)[https://codeclimate.com/github/bcgov/jag-file-submission]. Clicking on the badges found at the top of this projects README.md will also direct anyone to this page.
```
Comment (sdevalapurkar-bcgov) [inline docs/integration.md:10]:
```suggestion
View the JAG File Submission code quality report (here)[https://codeclimate.com/github/bcgov/jag-file-submission]. Clicking on the badges found at the top of this projects README.md will also redirect anyone to this page.
```

**Comment by akroon3r** [comment]
@sdevalapurkar-bcgov I've re-written a bunch of stuff, check it out again please

File: docs/integration.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# JAG File Submission Continuous Integration
+
+## Git Action Workflow
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://app.lucidchart.com/documents/embeddedchart/cd317d74-4127-4518-8ad9-b99e7a66e005" id="M9w_SX3Zmq.f"></iframe></div>
+
+## Code Climate
+
+The JAG File Submission project uses [Code Climate](https://codeclimate.com) as its code quality tool. Much like SonarQube, Code Climate reads and interprets the code coverage reports generated by both the frontend and backend components of this project, and then returns a Pass/Fail status back to Github. This is an attempt to move the code quality gate in this projects development lifecycle to a point where the Maintainability and Test Coverage scores are as high as possible before this application reaches a production ready state.
```
Comment (sdevalapurkar-bcgov) [inline docs/integration.md:9]: would be nice to say something about being run on pr before merge to master branch

Comment (akroon3r) [reply docs/integration.md:9]: @sdevalapurkar-bcgov again

**Comment by akroon3r** [comment]
@alexjoybc I marked the line length issues as won't fix as I'll be updating the code climate config to increase that.

the other issues have been resolved.

**Pull request merged**